Seminar3/ThirdSeminar/src/main/java/sopt/org/ThirdSeminar/controller/AlarmController.java

```diff
@@ -0,0 +1,36 @@
+package sopt.org.ThirdSeminar.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import sopt.org.ThirdSeminar.common.dto.ApiResponseDto;
+import sopt.org.ThirdSeminar.controller.dto.response.AlarmResponseDto;
+import sopt.org.ThirdSeminar.domain.Alarm;
+import sopt.org.ThirdSeminar.exception.SuccessStatus;
+import sopt.org.ThirdSeminar.service.AlarmService;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/alarms")
+public class AlarmController {
+    private final AlarmService alarmService;
+
+    // 알람 생성
+    @PostMapping("")
```

Comment:
```suggestion

    // 알람 생성
    @PostMapping
```
이런 상황에서 소괄호를 없애도 됩니다..!!